shared/actions/people.js

```diff
@@ -111,6 +111,17 @@ const _setupPeopleHandlers = () => {
     })
   })
 }
+
+const _onNavigateTo = (action: RouteTypes.NavigateAppend, state: TypedState) => {
+  const list = I.List(action.payload.path)
+  const root = list.first()
+  const peoplePath = getPath(state.routeTree.routeState, [peopleTab])
```

Comment:
Is this the path before the navigation happens or after?

Reply:
@buoyad After -- I don't think there's a way to access an old copy of the store from inside a saga, since they run after reducers.  Does that sound right?

Reply:
Yep that's good!